remail/email_api/service.py

```
@@ -0,0 +1,497 @@
+                to_recipients=item.to_recipients,
+                cc_recipients=[],
+                bcc_recipients=[],
+                date = parsed_datetime,
+                html_files=html
+                )]
+
+#-------------------------------------------------
+
+def imap_test():
+    imap = ImapProtocol()
+    test = Email(
```

Ruff (0.8.0)

Local variable `test` is assigned to but never used

Remove assignment to unused variable `test`

(F841)